woudc_data_registry/product/uv_index/uv_index_generator.py

```diff
@@ -80,9 +80,13 @@ def execute(path, update, start_year, end_year, bypass):
         registry_.session.query(UVIndex).delete()
         registry_.save()
 
+    previously_seen_instrument = []
     # traverse directory of files
-    for dataset_path in datasets:
-        for dirname, dirnames, filenames in os.walk(dataset_path):
+    for dataset in datasets:
+        successful_files = []
+        unsuccessful_files = []
+        for dirname, dirnames, filenames in os.walk(dataset):
```

Comment:
Likewise, rename `dataset` to `dataset_path` and the rest of the for loop block if applicable